src/sentry/static/sentry/app/stores/guideStore.jsx

```diff
@@ -18,12 +18,15 @@ const GuideStore = Reflux.createStore({
       // The current step of the current guide (1-indexed). 0 if there's no guide
       // or the guide is just cued but not opened.
       currentStep: 0,
+
+      isDrawerOpen: false,
```

Comment:
Since you're adding `isDrawerOpen` into the guide store, you should probably remove it from state in `helper.js` to avoid managing the same thing in 2 places. Also, this one isn't getting updated when the drawer closes, so it's not always correct.

Reply:
removed